src/headers.rs

```diff
@@ -364,6 +463,24 @@ mod tests {
             get_value(&header_map, "x-trino-session"),
             Some("a=1,b=4".to_string())
         );
+        assert_eq!(
+            get_value(&header_map, "x-trino-role"),
+            Some("moderator".to_string())
+        );
+        assert_eq!(
+            get_value(&header_map, "x-trino-transaction-id"),
+            Some("abc123".to_string())
+        );
+        
+        response_header_map.clear();
+        response_header_map.insert("X-Trino-Clear-Transaction-Id", "".parse().unwrap());
+
+        let mut request_headers = Headers::trino().transaction_id("abc123");
+        request_headers
+        .update_from_response_headers(&response_header_map)
```

Comment:
Did you run `cargo fmt` on this?  Indenting looks off.

Reply:
Oops, I've been using company tooling for a while and forgot about `cargo fmt`. Just ran it now and fixed the formatting.